Tasks/KubectlInstallerV0/src/utils.ts

```diff
@@ -0,0 +1,33 @@
+"use strict"
+
+import tl = require("azure-pipelines-task-lib/task");
+import toolLib = require("azure-pipelines-tool-lib/tool");
+import kubectlutility = require("utility-common/kubectlutility");
+
+export async function getKuberctlVersion(): Promise<string> {
+    let kubectlVersion = tl.getInput("kubectlVersion");
+    if(kubectlVersion) {        
+        if (kubectlVersion === "latest" || kubectlVersion === "")
```

Comment:
if (kubectlVersion === "latest") is enough.  If kubectlVersion  is empty  if(kubectlVersion) return false.